### PR TITLE
Add a few more job titles for some jobs.

### PR DESCRIPTION
**Pull request by Xymec**
## About The Pull Request
Adds some extra alternative job names for a few jobs.

## How This Contributes To The Skyrat Roleplay Experience
It doesn't but it's cool to have dripped out job names.

## Changelog

:cl:
add: Added more things
code: changed some code
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@
 	alt_titles = list("Detective", "Forensic Technician", "Private Investigator", "Forensic Scientist")
 
 /datum/job/doctor
-	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician")
+	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician", "Medical Intern")
```
Comment (tf-4) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: This is the only one I'm not really a fan of, intern roles I feel should be handled ICly and are more likely to be used as an excuse not to do work

Comment (Xymec) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Alright, I can fix that.

Comment (Xymec) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Alright, should be fixed now.

**Comment by Gandalf2k15** [comment]
fuck

**Comment by Gandalf2k15** [comment]
i didn't mean to merge that, /we

**Comment by Xymec** [comment]
ahahahha

**Pull request merged**